# HG changeset patch
# User John D
# Date 1490099564 14400
#      Tue Mar 21 08:32:44 2017 -0400
# Branch stable
# Node ID fc344de3afc9320d8fa5cd5b48428d81678c3725
# Parent  af5e8bfe03adbc46e510e45e1616b4309ea1ad91
Set version on AppUserModelId (Bug #50428)

* libinterp/corefcn/sysdep.cc
  (set_application_id): use gnu.octave.VERSION for SetCurrentProcessExplicitAppUserModelID

diff --git a/libinterp/corefcn/sysdep.cc b/libinterp/corefcn/sysdep.cc
--- a/libinterp/corefcn/sysdep.cc
+++ b/libinterp/corefcn/sysdep.cc
@@ -191,17 +191,17 @@ set_application_id (void)
 
   if (hShell != NULL)
     {
       SETCURRENTAPPID pfnSetCurrentProcessExplicitAppUserModelID =
         reinterpret_cast<SETCURRENTAPPID> (GetProcAddress (hShell,
                                            "SetCurrentProcessExplicitAppUserModelID"));
 
       if (pfnSetCurrentProcessExplicitAppUserModelID)
-        pfnSetCurrentProcessExplicitAppUserModelID (L"gnu.octave");
+        pfnSetCurrentProcessExplicitAppUserModelID (L"gnu.octave." VERSION);
 
       FreeLibrary (hShell);
     }
 
 #endif
 }
 
 DEFUN (__open_with_system_app__, args, ,
